### PR TITLE
docs(scanner): ADR for the NVD enricher

### PR DESCRIPTION
**Pull request by jvdm**
## Description

ADR to introduce the NVD enricher.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

N/A

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: scanner/decisions/0008-nvd-enricher.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Stackrox Scanner will have its own CVSS enricher that:
+
+1.  Pulls information from NVD API v2.
+2.  Pulls additional CVE information from NVD, leaving the door open to add or remove information as needed.
+3.  Replaces the NVD CVSS score pipeline by using the enricher in the vulnerability bundle exporter to consume the enrichments and vulnerability data later.
+
+## Consequences
+
+1.  Stackrox Scanner can decommission the NVD CVSS pipeline (workflows and Central handlers). 
+2.  Stackrox Scanner can programmatically add or remove additional NVD information to the security sources.  The enricher allows for filtering NVD data not used to reduce bundle sizes and increase them as needed.
```
Comment (RTann) [inline scanner/decisions/0008-nvd-enricher.md:31]: I was going to say it was nice that we didn't have to have yet another versioned data source: just the vuln data. However, this is a good point. With this approach, we filter out unnecessary data to decrease the data size

File: scanner/decisions/0008-nvd-enricher.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Decision
+
+Stackrox Scanner will have its own CVSS enricher that:
+
+1.  Pulls information from NVD API v2.
+2.  Pulls additional CVE information from NVD, leaving the door open to add or remove information as needed.
+3.  Replaces the NVD CVSS score pipeline by using the enricher in the vulnerability bundle exporter to consume the enrichments and vulnerability data later.
+
+## Consequences
```
Comment (RTann) [inline scanner/decisions/0008-nvd-enricher.md:28]: Do you have any numbers about what the size of the new .json.zst would be at this time?

**Review by RTann — APPROVED** [review]
LGTM. Just one question

File: scanner/decisions/0008-nvd-enricher.md (hunk 1 above)
Comment (jvdm) [reply scanner/decisions/0008-nvd-enricher.md:31]: As long as we have versioned data, the benefit of the data that is not versioned is negligible.

File: scanner/decisions/0008-nvd-enricher.md (hunk 2 above)
Comment (jvdm) [reply scanner/decisions/0008-nvd-enricher.md:28]: You can get an upper limit and a very good estimation for that number by adding the size of the current compressed NVD bundle generated in our workflow and adding to the current size of the current `.json.zst`.

Comment (RTann) [reply scanner/decisions/0008-nvd-enricher.md:28]: I was hoping we would have this datapoint in this doc, though

**Pull request merged**